themes/custom/az_barrio/az_barrio.theme

```diff
@@ -640,4 +640,16 @@ function az_barrio_preprocess_block(&$variables) {
       }
     }
   }
+
+/**
+ * Closes #2251 Remove block_class config foraz_barrio_search block and add
+ * az-search-block class via preprocess hook.
+ *
+ * @link https://github.com/az-digital/az_quickstart/pull/2252
```

Comment:
Personally, I think we could include just one link. I'd keep the issue link, but either one is definitely fine with me.

```suggestion
```